dev/ColorPicker/ColorPicker.cpp

```diff
@@ -301,6 +302,10 @@ void ColorPicker::OnPropertyChanged(winrt::DependencyPropertyChangedEventArgs co
     {
         OnColorSpectrumComponentsChanged(args);
     }
+    else if (property == s_OrientationProperty)
+    {
+        UpdateOrientation(unbox_value<winrt::Orientation>(args.NewValue()));
```

Comment:
I would unbox args.NewValue() in the UpdateOrientation function for consistency's sake.

Reply:
The thing is that we also want to be able to take an `Orientation` to update the orientation in `OnApplyTemplate` to be able to take the property set through XAML.

Reply:
UpdateOrientation will fire onLoad already, so you can actually remove it from OnApplyTemplate I'm pretty sure. You should also name the function to `OnOrientationChanged` for consistency's sake as well.

Reply:
Updated that now, if we don't need it than it's best to remove that code. Thank you for pointing that out.